eufy_security/camera.py

```diff
@@ -16,6 +19,21 @@ def __init__(self, api: "API", camera_info: dict) -> None:
         self._api = api
         self.camera_info: dict = camera_info
 
+    @property
+    def params(self):
+        # Parse camera info
+        params = {}
+        for param in self.camera_info["params"]:
+            param_type = param["param_type"]
+            value = param["param_value"]
+            try:
+                param_type = ParamType(param_type)
+                value = param_type.read_value(value)
+            except ValueError:
+                pass
```

Comment:
We should log a warning-level message here so that we can learn about unknown parameter types.

Reply:
Will add that. By default it wouldn't parse unknown parameters and simply return unchanged.